vendors/laiier/drivers/severn_wld/driver.yaml

```
@@ -0,0 +1,17 @@
+# Formal driver name
+name: Severn WLD Driver
+# Simple description of the driver functionalities
+description: Severn WLD decoding/encoding driver, following the LoRaWAN Payload Codec API specification.
+# Mandatory - id of the company that developed the driver
+# This id will be used to construct the id of the driver (Should be in lowercase, no special characters, maximum 8 characters)
+producerId: laiier
+# Email of the developer
+developerEmail: marina.finch@laiier.io
+# In case of a driver per model: `<vendorId>:<modelId>:<modelVersion>`
+# In case of a driver for multiples model: `<vendorId>:generic:<genericVersion>`
+# Needed for linking the model with a specific driver -> must be the same one used in model.yaml in the corresponding model
+# You should have only one protocolId per driver
+protocolId: laiier:severn_wld:1
```

"_" are not accepted in the protocolId
```suggestion
protocolId: laiier:severn-wld:1
```